docs/fedora-installation-guide.md

```diff
@@ -52,8 +52,8 @@ $ sudo systemctl start cc-proxy.socket
 
 5. Run Clear Containers 3.0.
 
-You are now ready to run Clear Containers 3.0. For example:
+   You are now ready to run Clear Containers 3.0. For example:
```

Comment:
Are we using "For example" here because there are multiple ways to run Clear Containers 3.0? If not, let's remove or replace "For example:"

Reply:
Agreed - branch updated.